go.mod

```diff
@@ -8,49 +8,49 @@ require (
 	github.com/go-logr/logr v1.2.4
 	github.com/onsi/ginkgo/v2 v2.13.0
 	github.com/onsi/gomega v1.28.0
-	github.com/operator-framework/api v0.17.4-0.20230223191600-0131a6301e42
-	github.com/operator-framework/catalogd v0.7.0
+	github.com/operator-framework/api v0.17.7
```

Comment:
I see that you updated to 0.17.7. Does it make sense to update to 0.18.0?

Reply:
This came in as a transitive dependency from operator-registry, but I'm happy to bump it to v0.18.0. Done [b10ddf6bf5](https://github.com/operator-framework/operator-controller/compare/1f2c1528b5a2696a867c502fc95222a181e34476..af4dfb7e5818909c84dbd8719d0f4bb10ddf6bf5)